src/main/java/io/connect/scylladb/ScyllaDbSessionImpl.java

```diff
@@ -67,6 +67,17 @@ public ResultSet executeQuery(String query) {
     return this.session.execute(query);
   }
 
+  @Override
+  public KeyspaceMetadata keyspaceMetadata(String keyspaceName){
+    log.trace("keyspaceMetadata() - keyspaceName = '{}'", keyspaceName);
+    return cluster.getMetadata().getKeyspace(keyspaceName);
```

Comment:
Do i need to check if metadata was explicitly set to disabled?
From .getKeyspace(keyspaceName) docs:
> Returns:
> the metadata of the requested keyspace or null if keyspace is not a known keyspace. Note that the result might be stale or null if metadata was explicitly disabled with QueryOptions.setMetadataEnabled(boolean).
@avelanarius

Reply:
We can assume that the metadata is enabled. For it to be disabled, the connector code would have to disable it explicitly and it doesn't do it and there's no reason to do it.